aiohttp/web_response.py

```diff
@@ -160,12 +163,7 @@ def set_status(
             not self.prepared
         ), "Cannot change the response status code after the headers have been sent"
         self._status = int(status)
-        if reason is None:
-            try:
-                reason = HTTPStatus(self._status).phrase
```

Comment:
So, the problem is the actual object being instantiated?

Reply:
Yes